tcms/xmlrpc/utils.py

```diff
@@ -12,14 +12,12 @@
 
 def parse_bool_value(value):
     if value in ACCEPTABLE_BOOL_VALUES:
-        if value is '0':
+        if value is '0':  # pylint: disable=literal-comparison
```

Comment:
What happens if you replace `is` with `==` ?

Reply:
Then pylint issues no warning. I have fixed that